### PR TITLE
Bug/mds 3369

### PR DESCRIPTION
**Pull request by mmumby**
- Added "last_updated_date" to the permitGenObject on both draft and issuance.
- Updated wording on the document
- 
Screen Shot 2021-01-25 at 1 07 04 PM
Screen Shot 2021-01-25 at 1 07 19 PM

### CODE REVIEW

File: services/core-web/src/components/noticeOfWork/applications/permitGeneration/NOWPermitGeneration.js [hunk 1]
```diff
@@ -200,6 +206,7 @@ export class NOWPermitGeneration extends Component {
       : amendment.regional_office;
     permitGenObject.now_tracking_number = noticeOfWork.now_tracking_number;
     permitGenObject.now_number = noticeOfWork.now_number;
+    // permitGenObject.application_last_updated_date = formatDate(noticeOfWork.last_updated_date);
```
Comment (nayr974) [inline services/core-web/src/components/noticeOfWork/applications/permitGeneration/NOWPermitGeneration.js:209]: Is this intentional? Can the line be removed or a note added as to why it's commented out temporarily if not?

**Review by nayr974 — COMMENTED** [review]
One question; looks good.

File: services/core-web/src/components/noticeOfWork/applications/permitGeneration/NOWPermitGeneration.js (hunk 1 above)
Comment (mmumby) [reply services/core-web/src/components/noticeOfWork/applications/permitGeneration/NOWPermitGeneration.js:209]: oh good catch, meant to delete.

Comment (mmumby) [reply services/core-web/src/components/noticeOfWork/applications/permitGeneration/NOWPermitGeneration.js]: removed!

**Review by nayr974 — APPROVED** [review]

**Pull request merged**